modules/internal/ChapelRange.chpl

```diff
@@ -344,6 +344,25 @@ module ChapelRange {
     compilerError("Bounds of 'low..high' must be integers of compatible types.");
   }
 
+  proc chpl__nudgeLowBound(low) {
+    return chpl__intToIdx(low.type, chpl__idxToInt(low) + 1);
+  }
+  proc chpl__nudgeHighBound(high) {
+    return chpl__intToIdx(high.type, chpl__idxToInt(high) - 1);
+  }
+  proc chpl__nudgeLowBound(param low) param {
+    return chpl__intToIdx(low.type, chpl__idxToInt(low) + 1);
+  }
```

Comment:
For the param version of `chpl__nudgeLowBound()`, please explain in a comment why you did not create an `integral` and `bool` versions while you created thouse for nudgeHighBound.

Reply:
In https://github.com/chapel-lang/chapel/pull/17460/commits/99d4ddd5fb0cd3b06558d7eb3c6614615875f0ea, I updated the comment on nudgeHighBound() to indicate why it needs to be different than nudgeLowBound() (since nudgeLowBound() is written the way we'd like it to be, so shouldn't raise questions in and of itself.  I expect to resolve this comment in the next week or so, though, to make both versions have a single generic param overload.